assets/scss/style.scss

```diff
@@ -0,0 +1,115 @@
+@use 'wiki.scss';
```

Comment:
My one concern here is that you used SCSS. What did you use to compile this to a regular CSS file? (Knowing what tool you used could enable us to use SCSS elsewhere and potentially automate the CSS compilation)